docs/meta-box.md

```diff
@@ -36,7 +70,7 @@ have to do, unless we want to move `createEditorInstance()` to fire in the foote
 or at some point after `admin_head`. With recent changes to editor bootstrapping
 this might now be possible. Test with ACF to make sure.
 
-### Redux and React Meta Box Management.
+### Redux and React Meta Box Management
 
 *The Redux store by default will hold all meta boxes as inactive*. When
 `INITIALIZE_META_BOX_STATE` comes in, the store will update any active meta box
```

Comment:
Unrelated but we should add this file to the Gutenberg Handbook.

Reply:
Good point, I didn't even notice that it wasn't there. 🙂 Added!